wavefront/values.yaml

```diff
@@ -163,6 +163,16 @@ proxy:
     tag: 6.1
     pullPolicy: IfNotPresent
 
+  ## Wavefront Collector resource requests and limits
+  ## Ref: http://kubernetes.io/docs/user-guide/compute-resources/
+  resources:
```

Comment:
The proxy typically requires much more than these. We'd want the values to be reasonable defaults for production environments. I'd suggest the following:

```
    requests:
      cpu: 100m
      memory: 1Gi
    limits:
      cpu: 1000m
      memory: 4Gi
```

Reply:
thanks for the quick response, updates pushed